### DISCUSSION TITLE
I want us to add Darkiworld on Immersive Translate please.

### DISCUSSION THREAD

**Discussion by Aslinjean** [immersive-translate/immersive-translate#1907] [General]
I don't know if you will take into account what I am about to say, but if it comes true, it would be one of the most beautiful gifts you could give me for my birthday on July 18th. I would like you to add https://www.darkiworld.com/ to your services so that I can watch movies/series with subtitles of my choice, as I am learning Spanish, but Spanish subtitles are not available on the official "Darkiworld" site. Therefore, since "Immersive Translate" is one of the best extensions on the market for subtitles, website translation, language learning, etc., I think that if "Immersive" supports "Darkiworld", I would be convinced that there is no better gift that could make me happier than this one. I don't know if I will receive a response to this, but if it were to happen, I guarantee to the whole team that I will be one of the always active users with a monthly subscription, not only because "Immersive" supports "Deepl" and "GPT-4", but also to support the whole team for these beautiful applications they put on the market. Good luck to me!

**Comment by Tipapaof** [comment]
I also voted for "Immersive" to support "Darkiworld".

**Comment by Ricardojarles** [comment]
I hope you find the answer you're looking for because it's not hard to obtain on "Immersive translate" since it's a website, I can say it's an alternative to "movie-web", so your dream can come true if the "Immersive" team wishes to help you.

**Comment by Drakechamp** [comment]
Thank you for introducing me to Darkiworld, it's a very nice website to watch movies/series without viruses.
